src/main/kotlin/de/livepoll/api/util/AccountListener.kt

```diff
@@ -29,15 +32,37 @@ class AccountListener: ApplicationListener<OnCreateAccountEvent> {
         accountService.createVerificationToken(user, token)
 
         val recipientAddress = user.email
-        val subject = "Account Confirmation Live-Poll"
-        val confirmationUrl = event.appUrl + "/v1/account/confirm?token=$token"
-        val message = "Please confirm your email: "
-
-        javaMailSender.send(SimpleMailMessage().apply {
-            setTo(recipientAddress)
-            setSubject(subject)
-            setText("$message\r\n$serverUrl$confirmationUrl")
-        })
-    }
+        val subject = "Your Live-Poll registration"
+        if(serverUrl.contains("api.")){
```

Comment:
I'd suggest to intruduce a new env variable here to inject the url of the frontend. We should keep our flexibility about hosting backend/frontend elsewhere without breaking it. Forthermore, this code would not work for the local dev environment, where we do not have `api.` in our urls.